File: mo-system/mo-system-api/src/main/java/ink/ckx/mo/system/api/model/entity/SysDept.java
package ink.ckx.mo.system.api.model.entity;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableId;
import ink.ckx.mo.common.core.base.BaseEntity;
import ink.ckx.mo.common.web.enums.StatusEnum;
import lombok.Data;

@Data
public class SysDept extends BaseEntity {

    @TableId(type = IdType.ASSIGN_ID)
    private Long id;

    private String name;

    private Long parentId;

    private String treePath;

    private Integer sort;

    private StatusEnum status;

    private Integer deleted;
}

File: mo-admin/mo-admin-api/src/main/kotlin/ink/ckx/mo/admin/api/model/entity/SysDept.kt
package ink.ckx.mo.admin.api.model.entity

import com.baomidou.mybatisplus.annotation.IdType
import com.baomidou.mybatisplus.annotation.TableId
import ink.ckx.mo.common.core.base.BaseEntity
import ink.ckx.mo.common.web.enums.StatusEnum

data class SysDept(
    @TableId(type = IdType.ASSIGN_ID)
    var id: Long? = null,
    var name: String? = null,
    var parentId: Long? = null,
    var treePath: String? = null,
    var sort: Int? = null,
    var status: StatusEnum? = null,
) : BaseEntity() 